doc/polyglossia.tex

```diff
@@ -942,90 +1010,120 @@ \subsection{finnish}\label{finnish}
-		Also note that this option is only functional (by design) if French is the main language.
-		\item \TB{frenchitemlabels}\new{v.1.46} = true or \textit{false} (default value = true). If \textit{true}, itemize item labels use em-dashes throughout,
-		as common in French typography.
-		Note that this option is only functional (by design) if French is the main language. Also, it might interfere with list packages such as \pkg{enumitem}.
-		\item \TB{itemlabels}\new{v.1.46} = ¦⟨cmd⟩¦ (default value = ¦\textemdash¦). If \emph{frenchitemlabels} is true, you can customize here the used item label of all levels.
-		\item \TB{itemlabeli}\new{v.1.46} = ¦⟨cmd⟩¦ (default value = ¦\textemdash¦). If \emph{frenchitemlabels} is true, you can customize here the used item label of the first level.
-		\item \TB{itemlabelii}\new{v.1.46} = ¦⟨cmd⟩¦ (default value = ¦\textemdash¦). If \emph{frenchitemlabels} is true, you can customize here the used item label of the second level.
-		\item \TB{itemlabeliii}\new{v.1.46} = ¦⟨cmd⟩¦ (default value = ¦\textemdash¦). If \emph{frenchitemlabels} is true, you can customize here the used item label of the third level.
-		\item \TB{itemlabeliv}\new{v.1.46} = ¦⟨cmd⟩¦ (default value = ¦\textemdash¦). If \emph{frenchitemlabels} is true, you can customize here the used item label of the fourth level.
+		\item \pgchoicekey{variant}{\pgdefault{french}, canadian (=~acadian)\new{1.45}, swiss\new{1.47}}
+			Currently, the only difference between the four variants is that ¦swiss¦
+			uses ¦thincolonspace=true¦ by default since this conforms to the Swiss
+			conventions.
+		\item \pgboolkeytrue{autospacing}
+			One of the most distinct features of French typography is the addition of
+			extra spacing around punctuation and quotation marks (guillemets). By
+			default, polyglossia adds these spaces automatically, so you don't need
+			to enter them. This options allows you to switch this feature off
+			globally.
+		\item \pgboolkeyfalse[1.46]{thincolonspace}
+			With ¦variant=swiss¦, the default value is ¦true¦. If ¦false¦, a full
+			(non-breaking) interword space is inserted before a colon. If ¦true¦, a
+			thinner space -- as before \texttt{;}, \texttt{!}, and \texttt{?} -- is
+			used. Note that this option must be set after the ¦variant¦ option.
+		\item \pgboolkeytrue{autospaceguillemets}[Up to version 1.44, the option was
+		called \textit{automaticspacesaroundguillemets}. For backwards compatibility
+		reasons, the more verbose old option is still supported.]
+			If you only want to disable the automatic addition of spacing after
+			opening and before closing guillemets (and not at punctuation), set this
+			to \textit{false}. Note that the more general option \textit{autospacing}
```

Comment:
For these, we should probably also use semantic markup

Reply:
That's right, but I would like to defer this to a future revision as it would require too much work for now.

Reply:
OK.